### PR TITLE
feat: Added community member profile for Adithya Krishna

### LINKED ISSUE #1804: [Community] Member Profile: Adithya Krishna
**Current Behavior**
@adithyaakrishna  has been a consistent contributor and community member. It's time to recognize his contributions by adding a member profile on the layer5.io/community/members page.

**Desired Situation**
Let's recognize @adithyaakrishna  as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: adithyaakrishna
- Twitter: adii_kris
- LinkedIn: adiiikris
- Link to profile picture: https://drive.google.com/file/d/1G50o5fIC6YtD5Bhakm1SLwyca_tRuZ72/view?usp=sharing
- Short Bio: TBA
- Location: Bengaluru, India
- Badge: `Landscape`

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
The CONTRIBUTING.md file has a "Common Types of Site Updates" section with detailed examples, which the contributor can follow here.

---

**Contributor Resources**
The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).

- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by OlaoluwaM**
**Description**
Added community member profile for Adithya Krishna

This PR fixes #1804

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/members/adithya-krishna/index.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Adithya Krishna
+position: Contributor
+image_path: ./adithya-krishna.jpeg
```
Comment (Tanuj22) [inline src/collections/members/adithya-krishna/index.mdx:4]: It should to be `jpg`. The build is failing probably due to this

**Comment by l5io** [comment]
🚀 Preview for commit c03a3d913aa17b1a4e064aa9fd26e0ef50e216e0 at: https://608aeda3f9c0ba349899a321--layer5ng.netlify.app

**Comment by OlaoluwaM** [comment]
Something seems to be the problem

**Comment by l5io** [comment]
🚀 Preview for commit 0d82e4b3f41532edc15cb0b91517a2c227b6ebd1 at: https://608af00222a3763bdf3e3bc9--layer5ng.netlify.app

**Comment by Tanuj22** [comment]
> Something seems to be the problem
The DCO was failing probably due to the editing of file directly from GitHub. I have flagged it to pass manually.

**Comment by Tanuj22** [comment]
> > Something seems to be the problem

The DCO was failing probably due to the editing of file directly from GitHub. I have flagged it to pass manually.

**Comment by adithyaakrishna** [comment]
@OlaoluwaM Thanks a lot for working on this 🥰

I guess there was this line to be added right, which was according to the content on the issue at #1804 
image

File: src/collections/members/adithya-krishna/index.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+name: Adithya Krishna
+position: Contributor
+image_path: ./adithya-krishna.jpg
+github: adithyaakrishna
+twitter: adii_kris
+linkedin: adiiikris
+location: Bengaluru, Karnataka, IN
+bio: An enthusiastic Final Year CSE student at Jyothy Institute of Technology, Bengaluru and trying to explore newer fields and a person who loves to contribute to Open Source Software. I'm also the type of person who says Yes! to everything spontaneously as I believe in utilizing every opportunity I can get and learning/upskilling myself in the mean process 👨‍💻
+blurb: Mesh Head
```
Comment (adithyaakrishna) [inline src/collections/members/adithya-krishna/index.mdx:10]: @Tanuj22 I guess there was this line to be added right, which was according to the content on the issue at #1804 

image

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]

File: src/collections/members/adithya-krishna/index.mdx (hunk 2 above)
Comment (Tanuj22) [reply src/collections/members/adithya-krishna/index.mdx:10]: Yes correct. @OlaoluwaM can you add it

Comment (OlaoluwaM) [reply src/collections/members/adithya-krishna/index.mdx:10]: Sure!

**Comment by l5io** [comment]
🚀 Preview for commit ef26f6aa4c643e3fad3f6de5604e9d5cce8f9899 at: https://608af95f06c99e374eff3bb0--layer5ng.netlify.app

**Review by adithyaakrishna — APPROVED** [review]

**Review by Tanuj22 — APPROVED** [review]
Thank you @OlaoluwaM . Congrats @adithyaakrishna :partying_face:

**Comment by adithyaakrishna** [comment]
> Thank you @OlaoluwaM . Congrats @adithyaakrishna 🥳

Thanks a lot @OlaoluwaM and @Tanuj22 🎉🥰

**Pull request merged**